client/app/components/onboardingGuide/GuideCoverPhotoPage.js

```diff
@@ -29,18 +30,25 @@ const GuideCoverPhotoPage = (props) => {
       }),
       div({
         className: css.infoTextContent,
-        dangerouslySetInnerHTML: { __html: t('advice') }, // eslint-disable-line react/no-danger
-      }),
+      }, [
+        t('web.admin.onboarding.guide.cover_photo.advice.content1',
+          { link: a({
+            href: 'http://support.sharetribe.com/knowledgebase/articles/744438',
+            target: '_blank',
+            alt: t('web.admin.onboarding.guide.cover_photo.advice.alt'),
+          }, t('web.admin.onboarding.guide.cover_photo.advice.link')) }),
+        br(),
+        t('web.admin.onboarding.guide.cover_photo.advice.content2', { width: 1920, height: 450 }),
```

Comment:
Those are ideal cover photo sized. They will be interpolated to the translation:

`"Note: the photo will be resized to %{width}x%{height} pixels. The top and bottom sections of images that are taller will be cropped."` becames `"Note: the photo will be resized to 1920x450 pixels. The top and bottom sections of images that are taller will be cropped."`

I think the dimensions should not be in the translation itself. If we for example decide to change the dimensions and they are in hard-coded in the translations, that would mean changing all the translations.